mlos_bench/mlos_bench/tests/config/environments/local/scripts/bench_run.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Helper script to run the benchmark and store
+the results and telemetry in CSV files.
+
+Run:
+    ./bench_run.py ./output-metrics.csv ./output-telemetry.csv`
+"""
```

Comment:
I think you need to run `make format` and checkin the changes for the CI to pass.

Reply:
fixed!